system/nxrecorder/nxrecorder_main.c

```diff
@@ -531,7 +531,7 @@ static int nxrecorder_cmd_help(FAR struct nxrecorder_s *precorder,
 
 int main(int argc, FAR char *argv[])
 {
-  char                    buffer[LINE_MAX];
+  FAR char               *buffer;
```

Comment:
revert the change

Reply:
memory save by using other stuff memory :D